source/agora/consensus/Validation.d

```diff
@@ -94,8 +94,8 @@ public string isInvalidReason (const Transaction tx, UTXOFinder findUTXO,
                 return "Transaction: Can only freeze a Payment transaction";
         }
 
-        if (sum_unspent != Amount.FreezeAmount)
```

Comment:
`Amount.FreezeAmount` should then be renamed to `Amount.MinFreezeAmount`, to make it obvious that it's the minimum freeze amount, not exact freeze amount.

Reply:
applied the suggest